contracts/vesting/VestingWalletRecovery.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-network/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.9;
+
+import "./VestingWalletV1.sol";
+
+/**
+ * This contract is desigend for recovering the in case the beneficiary was lost.
+ */
+contract VestingWalletRecovery is VestingWalletV1 {
+    event BeneficiaryUpdate(address newBeneficiary);
+
+    function updateBeneficiary(address newBeneficiary) external onlyOwner {
+        _setBeneficiary(newBeneficiary);
+        emit BeneficiaryUpdate(newBeneficiary);
+    }
```

Comment:
Would it make sense to add another function to accept the beneficiary change? That might be a way of avoiding an issue of changing the beneficiary to a centralized exchange wallet (as an example)

Reply:
> That might be a way of avoiding an issue of changing the beneficiary to a centralized exchange wallet

My understanding is that is exactly what they plan to use. Apparently, coinbase has a thing where the user as a "long lasting" address that can receive assets (including ERC20) ... but I'm not 100% sure it can be use to interract with arbitrary smart contracts.

Reply:
Right, I recall someone mention the Coinbase Account, but if that's the purpose then it is okay under the assumption that this VestingWallet is more legally-restricted than code-restricted.

Reply:
Yeah since `release` is permisionless we can't assume that the beneficiary wallet is able to interact with the smart contract.